002-routing-params-views/src/app/common/app.controller.js

```diff
@@ -0,0 +1,24 @@
+(function(){
+    'use strict';
+
+    angular
+        .module('common')
+        .controller('AppController', AppController);
+
+    /** @ngInject */
+    function AppController($state, $mdSidenav, Auth) {
+        var vm = this;
+        vm.user = Auth.getUser();
```

Comment:
If this is likely to change to a  server call put it in the resolve, also as this is a controller you can use vm.$onInit
(https://docs.angularjs.org/guide/component)